packages/docs/content/1_getting-started/2_using-components.mdx

```diff
@@ -0,0 +1,85 @@
+
+The examples below assume you've [installed the design system using NPM](/getting-started/installation/) and have already setup your build system for React.
+
+### Named imports
+
+Components can be imported from the package entry point using the syntax below.
+
+<Alert heading="Performance note" variation="warn" className="ds-u-margin-bottom--2 ds-u-font-size--small">
+   <p class="ds-c-alert__text">
+      This approach may result in a larger file size than intended. Depending on what module bundler you use, all of the design system's React components may be included in the bundled file even if you didn't specifically import them. This can be avoided by using <a href="https://webpack.js.org/guides/tree-shaking/">tree shaking in Webpack</a>.
+    </p>
+</Alert>
+
+```jsx
+import { Button, TextField } from '@cmsgov/design-system';
+```
+
+The `@cmsgov/design-system` package provides an ES modules version of our code to support Webpack 4's tree shaking optimizations. Our package is configured to use the `modules` property to let bundlers know to resolve imports from `@cmsgov/design-system` to the ES modules version when preferred. Read more about ESM in [Webpack's documentation](https://webpack.js.org/guides/author-libraries/#final-steps), or from [Rollup's proposal](https://github.com/rollup/rollup/wiki/pkg.module).
+
+### Individual imports
+
+Components can also be imported from their individual export file. This method carries no risk of including unused or extra code, as you are only importing from a specific file.
+
+```jsx
+import Button from '@cmsgov/design-system/dist/components/Button/Button';
+import TextField from '@cmsgov/design-system/dist/components/Button/TextField';
+```
+
+### Usage
+
+After importing the React component, tell React to render the component in the DOM:
+
+```jsx
+ReactDOM.render(
+  <Button
+    variation="primary"
+    className="custom-btn"
+    onClick={() => alert('Button clicked!')}>
+    Button react component
+  </Button>
+  document.querySelector('#app'),
+);
+```
+
+More React prop documentation, examples, and guidance can be found on component documentation pages.
+
+## Using CSS components
+
+If your project doesn't use React, we provide a HTML/CSS-only version of our components. This includes styles for every component in the library, but you’ll be responsible for writing the correct markup and updating classes and DOM attributes in response to user events.
+
+This example assumes you've already added [design system styles via CSS or Sass](/getting-started/sass-and-css/).
+
+```html
+<div class="ds-c-alert ds-c-alert--warn">
+  <div class="ds-c-alert__body">
+    <h3 class="ds-c-alert__heading">Warning status</h3>
+    <p class="ds-c-alert__text">Alert warning body text</p>
+  </div>
+</div>
+```
+
+More HTML/CSS examples and guidance can be found on component documentation pages.
+
+<h2 id="examples" class="ds-h2 ds-u-color--primary-darker">Examples</h2>
```

Comment:
Any idea why these two headings are special?

Reply:
It looks like the old page was this way, but I think we should probably normalize them. Not sure why they have the primary-darker color applied